internal/service/verify/resource_voice_phrase_content.go

```diff
@@ -0,0 +1,422 @@
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/patrickcping/pingone-go-sdk-v2/pingone/model"
+	"github.com/patrickcping/pingone-go-sdk-v2/verify"
+	"github.com/pingidentity/terraform-provider-pingone/internal/framework"
+	"github.com/pingidentity/terraform-provider-pingone/internal/sdk"
+	validation "github.com/pingidentity/terraform-provider-pingone/internal/verify"
+)
+
+// Types
+type VoicePhraseContentResource struct {
+	client *verify.APIClient
+	region model.RegionMapping
+}
+
+type voicePhraseContentResourceModel struct {
+	Id            types.String `tfsdk:"id"`
+	EnvironmentId types.String `tfsdk:"environment_id"`
+	VoicePhraseId types.String `tfsdk:"voice_phrase_id"`
+	Locale        types.String `tfsdk:"locale"`
+	Content       types.String `tfsdk:"content"`
+	CreatedAt     types.String `tfsdk:"created_at"`
+	UpdatedAt     types.String `tfsdk:"updated_at"`
+}
+
+// Framework interfaces
+var (
+	_ resource.Resource                = &VoicePhraseContentResource{}
+	_ resource.ResourceWithConfigure   = &VoicePhraseContentResource{}
+	_ resource.ResourceWithImportState = &VoicePhraseContentResource{}
+)
+
+// New Object
+func NewVoicePhraseContentResource() resource.Resource {
+	return &VoicePhraseContentResource{}
+}
+
+// Metadata
+func (r *VoicePhraseContentResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_voice_phrase_content"
+}
+
+func (r *VoicePhraseContentResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+
+	// schema descriptions and validation settings
+	const attrMinLength = 1
+
+	// P1 Platform does not set a traditional UUID as the default phrase ID value
+	const defaultVoicePhraseId = "exceptional_experiences"
+
+	phraseIdDescription := framework.SchemaAttributeDescriptionFromMarkdown(
+		"For a customer-defined phrase, the identifier (UUID) of the `voice_phrase` associated with the `voice_phrase_content` configuration. For pre-defined phrases, a string value.",
+	)
+
+	contentDescription := framework.SchemaAttributeDescriptionFromMarkdown(
+		"The phrase a user must speak as part of the voice enrollment or verification. The phrase must be written in the language and character set required by the language specified in the `locale` property.",
+	)
+
+	resp.Schema = schema.Schema{
+		// This description is used by the documentation generator and the language server.
+		Description: "Resource to configure the voice enrollment or verification requirements when configuring a `verify_policy` for voice verification.\n\n" +
```

Comment:
Is this description correct for the resource?

Reply:
Text was from API docs. Description has been reworded for clarity.